tests/libMarshal/UserRenamer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace libMarshal;
+
+use function ucfirst;
+use libMarshal\attributes\Field;
+use libMarshal\attributes\Renamer;
+
+#[Renamer("ucfirst")] class UserRenamer extends User {
+	use MarshalTrait;
+
+	/**
+	 * @param string $firstName
+	 * @param string $lastName
+	 * @param int $age
+	 * @param float $height
+	 * @param string[] $contacts
+	 * @param string|null $email
+	 */
+	public function __construct(
+		#[Field(name: "First-name")] public string $firstName,
```

Comment:
i know this part better developer make better part
but me nooooooooooooob no can better
pls help me

Reply:
What do you need help with?

Reply:
> What do you need help with?

code good tihing